.github/workflows/hadolint-analysis.yml

```diff
@@ -17,3 +17,10 @@ jobs:
       run: docker pull hadolint/hadolint
     - name: Lint Dockerfile
       run: docker run --rm --interactive hadolint/hadolint hadolint --ignore DL3018 --ignore DL4001 --ignore DL3013 --ignore SC2015 - < ./4/Dockerfile
+    - name: Notify failures on Slack
```

Comment:
i would remove this for a quality gate that will only trigger on PR and needs to pass prior to being able to merge. a dedicated slack notification does not seem necessary

Reply:
It makes sense, I'll remove it, thanks!